### PR TITLE
Remove unused file system api

### PR DESCRIPTION
**Pull request by anbraten**

### CODE REVIEW

File: server/store/datastore/migration/migration.go [hunk 1]
```diff
@@ -51,7 +51,6 @@ var allBeans = []interface{}{
 	new(model.Pipeline),
 	new(model.PipelineConfig),
 	new(model.Config),
-	new(model.File),
```
Comment (6543) [inline server/store/datastore/migration/migration.go:54]: should we drop that table too?

Comment (anbraten) [reply server/store/datastore/migration/migration.go:54]: Yes, I will create a migration. It is empty for all system as it wasn't used anyways.

**Comment by 6543** [comment]
@anbraten updated proto file for you

File: server/store/datastore/migration/migration.go (hunk 1 above)
Comment (anbraten) [reply server/store/datastore/migration/migration.go:54]: done

**Review by 6543 — APPROVED** [review]

**Pull request merged**